internal/pkg/term/selector/selector.go

```diff
@@ -163,13 +174,19 @@ type ConfigSelect struct {
 	workloadLister ConfigWorkloadLister
 }
 
-// WorkspaceSelect  is an application and environment selector, but can also choose a service from the workspace.
+// WorkspaceSelect  is an application and environment selector, but can also choose a service or pipeline from the workspace.
 type WorkspaceSelect struct {
 	*Select
 	ws      WorkspaceRetriever
 	appName string
 }
 
+// PipelineSelect is a workspace pipeline selector.
+type PipelineSelect struct {
+	*Select
```

Comment:
Do we need `*Select` or can we just get by with the following since `PipelineSelect` won't prompt for app or env names:
```suggestion
	prompt Prompter
```